### PR TITLE
Iosxe/show wireless stats ap join summary

### PR DESCRIPTION
**Pull request by robschn**

### CODE REVIEW

**Review by badalsoyantar — APPROVED** [review]

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Ap auth pending",
+            "failure_phase": "Join",
+            "ip_address": "10.10.194.10",
+            "status": "Joined",
+            "visitors_mac": "706d.05ec.fb60",
+        },
+        "706d.053b.0d20": {
+            "ap_name": "visitors-9105w",
+            "disconnect_reason": "Max Retransmission to " "AP",
```
Comment (robschn) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py:294]: I'll look into why the parser is adding `" "`

Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py]: I was using `pprint()` during testing and it added the extra `" "`. Fixed now! Waiting on review from @tahigash

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py [hunk 2]
```diff
@@ -0,0 +1,493 @@
+expected_output = {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py:1]: schema looks good

**Review by tahigash — APPROVED** [review]

**Comment by tahigash** [comment]
thank you for your contribution!

**Pull request merged**